### ISSUE TITLE
make Chpt 2 styling uniform via global scoping

### ISSUE THREAD

**Issue by ellenbechtel** [usgs-makerspace/temperature-prediction#58]
- [ ] loading of fonts and styles on different browsers
- [ ] make uniform chart title and annotation styles, put in globally scoped style tag

**Comment by ellenbechtel** [comment]
Edits from Hayley on 11/12

Nice, Ellen! Thanks for wrangling all of this css. It's really looking good.

Just going to note a few styling things as I notice them. Feel free to change these now or put them in an issue to be addressed later.

At some point we should update the color of the icons and associated text in ch1 (and those we add in ch2) to #6399ba, the same color as the rivers.

The 'algal' in Emergence of harmful algal blooms seems to run off screen on mobile?

This is pretty random, and is fine to leave as is if it's not a quick fix, but I noticed this upper stream segment appears thicker on mobile than on desktop?
image

I think the body text should be quite a bit smaller, both to distinguish it from the header text and to allow for a bit more negative space on the screen. Things feel a little crowded now (I'm having the same issues in the desktop ch2). I think it would also be good to have the body text match that in the svgs we inserted, so that the body text and our description of the matrix axes feel continuous. So matching the size and color of those two types of text would be great. And if the body text is grey, then our use of white to highlight important words/phrases will really stand out.

I think having some sort of header is a good idea. I think my issue with what we have currently is that we have two different header styles - the larger white font and the shadow-boxed white headers. So I think only using one of those two headers styles throughout would be good.

In the svg that explains the colors in the matrix, the 'in the dark' text looks super great now, but it is slightly overlapping the surrounding text.

I don't know that we need the header 'Observation Frequency for Reaches in the DRB'. I think the final availability matrix still fits within the previous section, which was titled 'Data Availability'. And I think this is the one spot where having another box in the midst of the content gets a bit busy visually.

The final data availability matrix box is slightly narrower than the other boxes that come before, where the matrix is being explained. I think we should either make the preceding boxes narrower, or remove the N-S axis on the final matrix so that it can be wider (though I think you were right to add it for clarity).

Also - the N-S axis line next to the availability matrix is a little hard to see, so it would be good to make that lighter if we keep it.
